install_config/install/advanced_install.adoc

```diff
@@ -1923,6 +2096,87 @@ ifdef::openshift-origin[]
 endif::[]
 ----
 
+[discrete]
+[[running-the-advanced-installation-tsb]]
+==== Optional: Deploying the Template Service Broker
+
+If you have xref:enabling-service-catalog[enabled the service catalog] and want
+to deploy the xref:configuring-template-service-broker[template service broker],
+run the following manual steps after the cluster installation completes
+successfully:
+
+[WARNING]
+====
+The template service broker (TSB) is currently a Technology Preview feature and
+should not be used in production clusters. Enabling the TSB currently requires
+opening unauthenticated access to the cluster; this security issue will be
+resolved before exiting the Technology Preview phase.
+====
```

Comment:
@bparees I can't remember if you ack'd the language in this warning box here, can you take a look?